code/modules/VR/vr_human.dm

```diff
@@ -0,0 +1,55 @@
+
+
+/mob/living/carbon/human/virtual_reality
+	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
+	var/obj/machinery/vr_sleeper/vr_sleeper
+	var/datum/action/quit_vr/quit_action
+
+
+/mob/living/carbon/human/virtual_reality/New()
+	..()
+	quit_action = new()
+	quit_action.Grant(src)
+
+
+/mob/living/carbon/human/virtual_reality/death()
+	revert_to_reality()
+	..()
```

Comment:
I'd recommend deleting the mob, lest we have a VR room with 50 dead bodies from someone entering and suiciding a billion times.